product_docs/docs/pgd/5.6/nodes/subscriber_only/optimizing-so.mdx

```diff
@@ -32,42 +32,44 @@ The write leader nodes in other non-subscriber-only groups replicate changes to
 
```

Comment:
In the para before this section head, I wanted to check whether the second sentence needed some additional punctuation or if there was some error, as it seemed to contradict itself.

Reply:
Have made changes to make it clearer.